apps/crn-frontend/src/network/teams/TeamProfile.tsx

```diff
@@ -56,15 +56,19 @@ const TeamProfile: FC<TeamProfileProps> = ({ currentTime }) => {
 
   const { pageSize } = usePaginationParams();
 
-  const options = getEventListOptions(
-    currentTime,
-    false,
-    {
-      pageSize,
-    },
-    { teamId },
-  );
-  const upcomingEventsResult = useEvents(options);
+  const getOptions = (isPast: boolean) =>
+    getEventListOptions(
+      currentTime,
+      isPast,
+      {
+        pageSize,
+      },
+      { teamId },
+    );
+
+  const upcomingEventsResult = useEvents(getOptions(false));
```

Comment:
not sure if it affects performance somewhere else, but `useEvents` could return an object with both upcoming events and past events